src/components/events/Slides.jsx

```diff
@@ -1,5 +1,38 @@
+import Image from "next/image";
+import { AiOutlineArrowLeft, AiOutlineArrowRight } from "react-icons/ai";
+
 const Slides = () => {
-  return <div>Slides</div>;
+  return (
+    <div className="Slides">
+      <div className="flex justify-center items-center">
+        <div className="my-55 mr-40">
+          <AiOutlineArrowLeft
+            size={50}
+            className="text-asme-blue-600 cursor-pointer text-xl"
+          />
+        </div>
+
+        <div className="center section">
+          <div className="font-playfair text-lg pt-4">
+            <p>Event Name</p>
+          </div>
+          <Image
+            src="registration-line.png" // Path to your image inside the 'public' directory
```

Comment:
import as an object instead of specifying width and height

i.e. something like
import RegistrationLine from registration-line.png
...
<Image
src={RegistrationLine}